extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1056,6 +1056,20 @@ describes.realWin(
         });
       });
 
+      it('should add or remove the muted attribute on muted state change', async () => {
+        await createStoryWithPages(2, ['cover', 'page-1']);
+
+        await story.layoutCallback();
+
+        expect(story.element.hasAttribute('muted')).to.be.true;
+
+        story.storeService_.dispatch(Action.TOGGLE_MUTED, false);
+        expect(story.element.hasAttribute('muted')).to.be.false;
+
+        story.storeService_.dispatch(Action.TOGGLE_MUTED, true);
+        expect(story.element.hasAttribute('muted')).to.be.true;
```

Comment:
Plz split this test in two so one test = one assertion. It's a good practice that makes it easier to figure out what's broken when a test fail.